.github/workflows/emberjs-app-lint.yml

```diff
@@ -11,6 +10,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v2
         with:
+          #TODO: Verify Node version
```

Comment:
Each place where a value would be expected to change from project to project has now been tagged with a `#TODO` comment.